src/services/bindService.js

```diff
@@ -206,6 +206,8 @@ angular.module("openshiftCommonServices")
     return {
       bindingResource: bindingResource,
       getServiceClassForInstance: getServiceClassForInstance,
+      makeParametersSecret: makeParametersSecret,
```

Comment:
Change the `binding` parameter for this method to `owner` since it can be an instance too now?

Reply:
makes sense